pkgs/development/tools/gettext-tools/default.nix

```diff
@@ -0,0 +1,105 @@
+{ stdenv, lib, fetchurl, libiconv, xz, gettext-runtime, libtextstyle,
+  autoreconfHook }:
+
+
+let common = import ../../libraries/gettext/common.nix { inherit fetchurl; };
+in
+
+# This assertion is only here to ensure that at the next update the following
+# tasks are performed:
+#   - remove the restore_the_ability_to_build_gettext-tools_separately.patch
+#     backport patch
+#   - the autoreconfHook
+#   - remove the 'cp ../libtextstyle/m4/libtextstyle.m4 gnulib-m4/' from postPach
+#
+# All those were nessary to backport
+# https://git.savannah.gnu.org/gitweb/?p=gettext.git;a=commitdiff;h=e4b3a3f56fa6fc2a51769e286545f0631bb4837c
+# This is necessry because of https://savannah.gnu.org/bugs/?56333
+assert common.version == "0.20.1";
```

Comment:
No need to add assert. The patch failing to apply should be sufficient heads-up. The comment can be converted to `TODO` next to the patch.